tools/twix/src/repository_parameters.rs

```diff
@@ -32,46 +32,61 @@ impl RepositoryParameters {
 
     pub fn write(&self, address: &str, path: String, value: Value) {
         let repository = self.repository.clone();
-        let Ok(head_id) = self
-            .head_id_from_address(address)
+        let Ok(hardware_ids) = self
+            .hardware_ids_from_address(address)
         else {
             error!("failed to get head ID from address {address}");
             return
         };
         self.runtime.spawn(async move {
-            let mut stored_value = repository
-                .read_configuration(&head_id)
+            let stored_complete_parameter_tree: Value = parameters::directory::deserialize(
+                repository.root_directory(),
```

Comment:
Is this root directory correct? Shouldn't this be

```suggestion
                repository.root_directory().join("etc/configuration"),
```

?

Reply:
I'll need to double check, as my memory serves deserialize takes the repo root.

Reply:
https://github.com/HULKs/hulk/blob/main/crates/communication/src/server/parameters/directory.rs#L54

Reply:
Damnit! you are right, I mixed up the two as repository crate and this have different meanings.
I renamed `root_path` to `parameters_root_path` for directory.rs. Hope it is not a problem.

I made the existing `configuration_root` method of repository crate public and used it.